src/views/ErrorView.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
Is this an `ErrorView` or an `NotFound` view? maybe we'd want to have a different view that renders general errors?